test/tests.js

```diff
@@ -173,4 +227,30 @@ describe('OneTrust Forwarder', function() {
             done();
         });
     });
+    
+    it('should use the proper regulation if consent purpose is "data_sale_opt_out"', function (done) {
```

Comment:
nit:

```suggestion
    it('should use CCPA regulation if consent purpose is "data_sale_opt_out", otherwise should use GDPR', function (done) {
```